wpedantic results in "extra ;" warning

```
[build] ../json-schema-validator/src/json-patch.cpp:69:2: error: extra ‘;’ [-Wpedantic]
[build]    69 | }; // namespace
[build]       |  ^
```

--- a/src/json-patch.cpp
+++ b/src/json-patch.cpp
@@ -66,7 +66,7 @@ const nlohmann::json patch_schema = R"patch({
         }
     }
 })patch"_json;
-}; // namespace
+} // namespace
 
 namespace nlohmann
 {

